### PR TITLE
docs(kusionstack.io/docs): add git guideline

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [x] N
- [ ] Y 

#### 2. What is the scope of this PR (e.g. component or file name):

/Users/shijun/Workspace/kusionstack.io/docs/governance/contribute/git-guideline.md

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

add git-guideline.md to show more details about naming branch, commit message and CLA.

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: docs/governance/contribute/git-guideline.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Git 提交指南
+
+本文介绍了Git提交变更时需要注意的事项，如果拒绝接受本文的内容会导致提交的变更无法被接受。
```
Comment (Peefy) [inline docs/governance/contribute/git-guideline.md:3]: Small tip: Chinese and English generally have a small space, you can use this markdown formatting tool: https://github.com/elliotxx/mdfmt

**Review by Peefy — COMMENTED** [review]
Better to have a English ver.

File: docs/governance/contribute/git-guideline.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+- docs:     -- 进行了文档部分的修改。
+- style:    -- 对代码格式的修改，并不影响代码的功能，如：删除多余空格，代码缩进等。
+- refactor: -- 在不改变代码功能的基础上对代码进行了的重构。
+- perf:     -- 对代码进行了性能优化。
+- test:     -- 添加或者调整已有的测试用例。
+- build:    -- 对构建系统或者外部依赖库进行了调整。
+- ci:       -- 调整了CI的配置文件或者脚本。
+- chore:    -- 对源代码和测试文件之外其他部分的调整。
+- revert:   -- 对commit进行回滚。
+```
```
Comment (elliotxx) [inline docs/governance/contribute/git-guideline.md:106]: Small tip: Suggestion is attached to the reference link:
[Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines)

File: docs/governance/contribute/git-guideline.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+ - {a_short_description}: 一个简短的描述，介绍这个分支的主要内容。
+
+ e.g. 张三创建了一个修复了输出格式化bug的分支，并打算使用账户 zhangS 提交 mr，对应的分支名称`fix/zhangS/fix_output_fmt_bug`。
+
+## 3. 关于Git Commit
+我们参考[Commitizen](https://github.com/commitizen/cz-cli)书写Commit Message。
+```
+注: 如果直接使用Commitizen生成Commit Message，需要注意因为Commitizen是开发人员
+管理commit的工具，与项目本身无关联，因此由Commitizen生成的中间产物(如: node_modules文件
+目录)可能没有在项目.gitignore文件中。
```
Comment (He1pa) [inline docs/governance/contribute/git-guideline.md:35]: You can give the solution, e.g. , "If you want to create a commit message by commitizen, add the following code snippet to .gitignore".
```
# commitizen
package.json
package-lock.json
node_modules/*
```

**Review by chai2010 — APPROVED** [review]
LGTM

**Pull request merged**